File: Megaminds_prectical.Model/UserModel.cs
namespace Megaminds_prectical.Model
{
    public static class Common
    {
        public static string ConnectionString = "";
    }
    public class AddUserModel
    {
        public string username { get; set; }
        public string email { get; set; }
        public string mobileNo { get; set; }
        public string address { get; set; }
        public int stateId { get; set; }
        public int? id { get; set; }
        public int cityId { get; set; }
    }

    public class StateModel
    {
        public int Id { get; set; }
        public string Statename { get; set; }
    }


    public class CityModel
    {
        public int Id { get; set; }
        public int StateId { get; set; }
        public string Cityname { get; set; }
    }

    public class UserListModel
    {
        public int Id { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
        public string MobileNo { get; set; }
        public string Address { get; set; }
        public int StateId { get; set; }
        public int CityId { get; set; }
        public string Cityname { get; set; }
        public string Statename { get; set; }
    }
    public class TranStatus
    {
        public int Code { get; set; } = 0;
        public string RetrunMassage { get; set; } = "";
    }
}

File: Megaminds_prectical/ClientApp/src/app/Modal/usermodel.ts
export class UserModel {
    username: string;
    email: string;
    mobileNo: string;
    address: string;
    cityId: number;
    stateId: number;
    id?: number;
}
export class StateListModel {
    id: number;
    statename: string;
}
export class CityListModel {
    id: number;
    stateId: number;
    cityname: string;
}

export class UserListModel {
    username: string;
    email: string;
    mobileNo: string;
    acityname: string;
    statename: string;
    cityId: number;
    stateId: number;
}